src/Rule/MultiplyAndDivideByStringRector.php

```diff
@@ -31,7 +32,7 @@ final class MultiplyAndDivideByStringRector extends AbstractRector implements Al
 {
     public const PRECISION = 'precision';
 
-    private int $precision;
+    private int $precision = 5;
```

Comment:
This change is not necessary after 0e7e9367 because `config/sets/v4.php` was improved - `MultiplyAndDivideByStringRector` is configured by default, so this value is set through `configure()`. Yeah, it's still possible to include rule without config (adding standalone rule, not set) and it will fail with uninitialised property, but since your PR does not relate to this, I would recommend to revert this change. I like to not mix things 😉

Reply:
I actually had an "uninitialised access" being thrown when I run the rector on the code base with `dev-main`. But yes, shouldnt be part of this PR.